ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/_cq_design_dialog/.content.xml

```diff
@@ -0,0 +1,138 @@
+                        jcr:title="Container Settings"
+                        sling:resourceType="granite/ui/components/coral/foundation/container"
+                        margin="{Boolean}true">
+                        <items jcr:primaryType="nt:unstructured">
+                            <container
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/container">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <type
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                                        fieldLabel="Layout"
+                                        name="./layout">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <default
+                                                jcr:primaryType="nt:unstructured"
+                                                text="(Default)"
+                                                value=""/>
+                                            <simple
+                                                jcr:primaryType="nt:unstructured"
+                                                text="Simple"
+                                                value="simple"/>
+                                            <responsive
+                                                jcr:primaryType="nt:unstructured"
+                                                text="Responsive Grid"
+                                                value="responsiveGrid"/>
+                                        </items>
+                                    </type>
+                                    <disableLayout
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                        fieldDescription="Check to disable layout selection in edit dialog."
+                                        name="./layoutDisabled"
+                                        text="Disable Layout"
+                                        checked="{Boolean}false"
+                                        uncheckedValue="false"
+                                        value="true"/>
+                                    <backgroundImageLabel
+                                        granite:class="coral-Form-fieldlabel"
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/text"
+                                        text="Background Image"/>
```

Comment:
are we keeping these properties in dialog ?

Reply:
we inherit these from sites container, i don't see why we need to loose these.